packages/sit-onyx/src/components/OnyxNavItem/OnyxNavItem.vue

```diff
@@ -0,0 +1,114 @@
+<script lang="ts" setup>
+import { computed } from "vue";
+import type { OnyxNavItemProps } from "./types";
+import OnyxListbox from "../OnyxListbox/OnyxListbox.vue";
+import type { ListboxOption } from "../OnyxListbox/types";
+
+const props = withDefaults(defineProps<OnyxNavItemProps>(), {
+  active: false,
+});
+
+const emit = defineEmits<{
+  /**
+   * Emitted when the nav item is clicked (via click or keyboard).
+   */
+  navigate: [href: string];
+}>();
+
+defineSlots<{
+  /**
+   * An optional slot to show additional content behind the label (e.g. a `OnyxBadge`).
+   */
+  default?(): unknown;
```

Comment:
Do you also want to add a Storybook example, test and screenshot for a nav item with slot content?